packages/client/browser/libp2pnode.ts

```diff
@@ -3,16 +3,14 @@
  * @memberof module:net/peer
  */
 
-import LibP2p from 'libp2p'
 import { NOISE } from '@chainsafe/libp2p-noise'
+import LibP2p from 'libp2p'
 import PeerId from 'peer-id'
 import { Multiaddr } from 'multiaddr'
-// types currently unavailable for below libp2p deps,
-// tracking issue: https://github.com/libp2p/js-libp2p/issues/659
-const LibP2pWebsockets = require('libp2p-websockets')
```

Comment:
Just out of interest (without further digging in): the referenced issue here https://github.com/libp2p/js-libp2p/issues/659 is still open, could this nevertheless be resolved?

Reply:
I was able to convert a few of the modules to import, the rest are still on require, I think it's obvious at this point that any modules using require are due to missing types, so I thought I'd clean up/remove the comment.